.travis.yml

```diff
@@ -34,9 +34,5 @@ before_script:
 script:
   - "make test"
   - "python manage.py makemessages -l eo"
-branches:
-    only:
-      - master
-      - ora-staff-grading
```

Comment:
For my own edification - what's the impact of this change? Just that Travis builds will run more regularly on non-master branches?